build-hugo.sh

```
@@ -0,0 +1,31 @@
+
+echo 'export PATH=$PATH:$GOBIN' >> ~/.bashrc
+echo 'export PATH=$PATH:$GOBIN' >> ~/.zshrc
+
+echo 'export PATH=$PATH:$HOME/go/bin' >> ~/.bashrc
+echo 'export PATH=$PATH:$HOME/go/bin' >> ~/.zshrc
+
+# Source the bashrc or zshrc file (depending on the shell)
+source ~/.bashrc
+
+# Verify installation
+if ! command -v $HOME/go/bin/hugo &> /dev/null
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)